app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,49 @@
+class ReviewsController < ApplicationController
+  #might only need index show delete
+  def index #show all
+    #@review = Review.where(product_id: params[:product_id]).first
+
+  end
+
+  def show # show one all display a view that has already been created
+      @review = Review.all.where(params[:product_id])
+      render :new
+  end
+
+  def new
+      @review = Review.new
+      @product = Product.find(params[:product_id])
+      render :new
```

Comment:
`render :new` is implicit in this case, so you could take this out :) Rails will by default render the view with the same name as the method in the corresponding views directory